### ISSUE TITLE
updateEvent, update complex attributes

### ISSUE THREAD

**Issue by xlthor** [fullcalendar/fullcalendar#2864]
due to the "isAtomic" function, event attributes of type Array get not updated to an event, although these get populated to the events when coming from an Ajax call.

Changed isAtomic:

```
  // Is the given value a non-object non-function value?
 function isAtomic(val) {
  var t = jQuery.type(val);
  return (/undefined|null|boolean|number|string|array/.test(t));
 }
```

**Comment by arshaw** [comment]
the reason i originally did this was because i didn't want event object sharing references to objects (could result in strange side effects) but i do agree that there should be a way to do this

**Comment by LeoFlandin** [comment]
I agree, I stored a array of strings in the Event but it is no possible to update it. It's a pity.

**Comment by arshaw** [comment]
queued up for release
https://github.com/fullcalendar/fullcalendar/commit/27aa278e6d9d91a91ea0a3186f842d26e6b53c97#diff-39a7189a16fb04482e74089e81ba1f02L559

**Comment by arshaw** [comment]
Released in [v3.6.0](https://github.com/fullcalendar/fullcalendar/releases/tag/v3.6.0)
